scammer-label-propagation/src/agent.py

```diff
@@ -0,0 +1,149 @@
+import logging
+import time
+import forta_agent
+from web3 import Web3
+from forta_agent import get_json_rpc_url, Finding
+from concurrent.futures import ThreadPoolExecutor
+
+from src.main import run_all
+from src.constants import attacker_bots, ATTACKER_CONFIDENCE, N_WORKERS, CHAIN_ID, MAX_FINDINGS
+
+logging.basicConfig(filename=f"logs.log", level=logging.INFO, 
+                    format='%(levelname)s:%(asctime)s:%(name)s:%(lineno)d:%(message)s')
+logger = logging.getLogger(__name__)
+
+
+def run_all_extended(central_node, alert_id):
+    try:
+        attackers_df = run_all(central_node)
+    except Exception as e:
+        logger.error(f"{central_node}:\tError running run_all in a thread: {e}")
+        return []
+    # Now we put things into a list of findings
+    all_findings_list = []
+    finding_dict = {
+            'name': 'scammer-label-propagation',
+            'description': 'Address marked as scammer by label propagation',
+            'alert_id': 'SCAMMER-LABEL-PROPAGATION-1',
+            'severity': forta_agent.FindingSeverity.High,
+            'type': forta_agent.FindingType.Scam
```

Comment:
is there any additional metadata you want to set that helps you understand the alert better (I am thinking statistics of the graph?) This way, once we have alerts labeled into TP/FP, you may be able to refine the thresholds of this bot.

Reply:
Also, you probably want to pass in the alert information from the alert/label that was passed into this bot. This way we can build a chain from label to alert to label to alert.

Reply:
Added information in metadata about original alert, and graph information